src/game/game.c

```diff
@@ -13,6 +13,17 @@
 
 #include <SDL2/SDL_image.h>
 
+static void set_window_title(struct oshu_game **game)
+{
+	const char * title = "Oshu! ♫ \"%s\" by %s";
```

Comment:
Please attach the star to the variable name.

Reply:
I believe writing the string literal inside the asprintf call is clearer than using a temporary variable, and compilers are really good at detecting format string errors when you don't make their job harder with intermediate variables.

Reply:
Shouldn't we use a define here  ? ;)